components/ui/ProjectCard.vue

```diff
@@ -1,103 +1,109 @@
 <template>
   <article class="project-card">
-    <div class="icon">
-      <nuxt-link v-if="isModrinth" :to="'/mod/' + id">
-        <img
-          :src="
-            iconUrl
-              ? iconUrl
-              : 'https://cdn.modrinth.com/placeholder.svg?inline'
-          "
-          :alt="name"
-        />
-      </nuxt-link>
-    </div>
-    <div class="info">
-      <div class="top">
-        <h2 class="title">
-          <nuxt-link v-if="isModrinth" :to="'/mod/' + id">{{ name }}</nuxt-link>
-          <a v-else :href="pageUrl">{{ name }}</a>
-        </h2>
-        <p v-if="author" class="author">
-          by <nuxt-link :to="'/user/' + author">{{ author }}</nuxt-link>
-        </p>
+    <div class="columns">
+      <div class="icon">
+        <nuxt-link v-if="isModrinth" :to="'/mod/' + id">
+          <img
+            :src="
+              iconUrl
+                ? iconUrl
```

Comment:
Use the ``Logical OR (||)`` operator.

Reply:
Or the ES6 ??, more stable and not relying on some js trickery

Reply:
> Or the ES6 ??, more stable and not relying on some js trickery

In this case it'd be better to use ``||`` as values like an empty string should just default to the placeholder icon, with ``??`` it  would not.